Cargo.toml

```diff
@@ -188,3 +188,17 @@ large_futures = "warn"
 [workspace.lints.rust]
 unexpected_cfgs = { level = "warn", check-cfg = ["cfg(tarpaulin)"] }
 unused_qualifications = "deny"
+
+[patch.crates-io]
+arrow = { git = "https://github.com/apache/arrow-rs.git", rev = "b1bfa0f5852536ef2edb43ac8ddb66c39e8a7233" }
```

Comment:
Oh arrow-rs doesn't plan to publish the official patch?

Reply:
Yeah -- it isn't officially released yet (I expect it to be done in a day or two)
- https://github.com/apache/arrow-rs/issues/6929

Sorry -- I should have maked this PR as draft